.github/workflows/CI.yml

```diff
@@ -110,3 +111,14 @@ jobs:
           ./node_modules/.bin/nightwatch -c nightwatch.js --env edge40,edge44
           ./node_modules/.bin/nightwatch -c nightwatch.js --env firefox45,chrome58,ie11
           pkill node || echo "Node process not running (anymore)"
+
+  # Tests the list of offline files in the Service Worker and compares it to the contents of the www directory
+  test-offline-files:
+    # We want to run on external PRs, but not on our own internal PRs as they'll be run by the push to the branch
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: windows-latest
```

Comment:
In the short term, it's not a problem to run this script on Windows.
In the mid or long term, it might be : if one day we switch to another CI that would only support linux, and/or if we want to avoid having to pay a Windows license.

I understand you're more comfortable with PowerShell than with Bash, so don't rewrite this script. It should be easy to rewrite later if necessary.

NB : I know PowerShell can run on Linux, but it's not commonly installed on linux distros

Reply:
I understand. I didn't feel I knew enough bash to be comfortable getting things right (or it would have taken a very long time and a lot of testing, trial-and-error). I'm sure you could rewrite it based on the same logic and regexes in about 10 minutes, but as you say, that can wait till we need to do it (hopefully not!).